app/src/main/java/org/p2p/wallet/auth/repository/CountryCodeInMemoryRepository.kt

```diff
@@ -66,6 +67,13 @@ class CountryCodeInMemoryRepository(
     override fun isValidNumberForRegion(phoneNumber: String, regionCode: String): Boolean =
         countryCodeHelper.isValidNumberForRegion(phoneNumber, regionCode)
 
+    override fun parsePhoneNumber(number: String, defaultRegionAlpha2: String): Pair<CountryCode, String>? {
```

Comment:
The PhoneNumber contains only single field: formatterPhoneNumber, but here phone number is a national one, without country code at the beginning.
But maybe, it would better to create separate model such as InternationalPhoneNumber or PhoneNumberWithCode where we would have 2 fields: country code and national number. What do you think?